app/models/post.rb

```diff
@@ -1,2 +1,14 @@
 class Post < ApplicationRecord
+  has_many :likes, foreign_key: post_id
```

Comment:
- [ ] There's a typo, Please fix it. Replace `post_id` with `:post_id` or `'post_id'`